docs/blog/2021-12-12-log4j-zero-day-mitigation-guide.mdx

```diff
@@ -48,29 +48,29 @@ earlier blog post](https://www.lunasec.io/docs/blog/log4j-zero-day/).
 Because of the severe impact from this vulnerability, there has been a lot of discussion on the internet
 about it. **Some of this information is outdated or wrong and _will_ leave you vulnerable if you follow it!**
 
-In contrast, this guide has been written by a team of professional Security Engineers at LunaSec. Everything here has
-been peer-reviewed by multiple security experts, and where possible our sources will be linked for other Security
+By contrast, this guide has been written by a team of professional Security Engineers at LunaSec. Everything here has
+been peer-reviewed by multiple security experts, and where possible our sources will be linked from other Security
 professionals to verify against.  This post links to many other guides and how-tos that we believe are trustworthy.  
 
 The full list of common bad advice is at the [bottom of this post](#known-bad-advice). If you believe you've already 
-mitigated Log4Shell, or you believe you're not vulnerable, please double-check your current information is up-to-date. 
+mitigated Log4Shell, or you believe you're not vulnerable, please double-check that your current information is up-to-date. 
 
 ## Determine if you are impacted by Log4Shell
 
-This vulnerability affects anybody who's using the log4j packages. That means it's
-primarily Java, but other languages like Scala, Groovy, or Clojure are also impacted.
+This vulnerability affects anybody who's using the log4j packages. That means that
```

Comment:
"that means primarily java...."

Reply:
You are right, this feels a little redundant.  This sounds kinda fucky too though, I went with an alternative that's a little more complex as a sentence but reads a little better and clearer IMO.  If you hate it I'll just revert.